riscv/processor.cc

```diff
@@ -922,11 +942,25 @@ void processor_t::set_csr(int which, reg_t val)
       VU.vxsat = (val & VCSR_VXSAT) >> VCSR_VXSAT_SHIFT;
       VU.vxrm = (val & VCSR_VXRM) >> VCSR_VXRM_SHIFT;
       break;
-    case CSR_MSTATUS: {
+    case CSR_MSTATUSH:
+      {
+	reg_t mask = MSTATUS_SBE | MSTATUS_MBE;
+	if (xlen != 32)
+	  break;
+	val = (state.mstatus & ~mask) | ((val << 32) & mask);
+      }
+      if (0) {
+	/* FALLTHROUGH */
+    case CSR_MSTATUS:
+        if (xlen == 32)
+	  val = ((state.mstatus >> 32) << 32) | (uint32_t)val;
```

Comment:
Indenting

Reply:
Not sure what you would like here.  The code is inside an `if()` block (even though it is reached without executing the actual `if`statement), so I indented two steps for that, which is what the codebase does in other places with `if`s.  Is it wrong here?

Reply:
Line 956 is inside the if() of line 955 so it should be indented more than line 955

Reply:
Ah, it is actually, but there is an illusion in the patch due to line 956 using a TAB character first on the line.  I'll run `M-x untabify` on the file to sort it out.